base/shell.jl

```diff
@@ -81,7 +81,7 @@ function shell_parse(raw::AbstractString, interp::Bool)
                     if done(s,k)
                         error("unterminated double quote")
                     end
-                    if s[k] == '"' || s[k] == '$'
+                    if s[k] == '"' || s[k] == '$' || s[k] == '\\'
```

Comment:
I was talking about the string `"\\"`, which is represented as `"\"\\\\\""` when input as Julia source. This string should lead to a single backslash. See e.g. in a shell:
```sh
echo "\\"
\
```
The first backslash escapes the second (this still happens inside double quotes), and `shell_parse` strips quotes around arguments.

In Julia:
```Julia
str="\"\\\\\""
println(str)
Base.shell_split(str)
["\\"]
```